lib/scsi_defs.h

```diff
@@ -0,0 +1,154 @@
+#ifndef LIB_SCSI_DEFS_H
+#define LIB_SCSI_DEFS_H
+
+namespace scsi_defs {
+
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 2.4.1 Table 11
+  enum SenseResponse {
+    CURRENT_DESCRIPTOR_ERROR = 0x72,
+    DEFERRED_DESCRIPTOR_ERROR = 0x73,
+    CURRENT_FIXED_ERROR = 0x70,
+    DEFERRED_FIXED_ERROR = 0x71
+  };
+
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 2.4.1.5 Table 28
+  enum SenseKey {
+    NO_SENSE = 0x0,
+    RECOVERED_ERROR = 0x1,
+    NOT_READY = 0x2,
```

Comment:
Please follow the same style as other PR. (enum class and kNoSense instead of NO_SENSE).

Reply:
Updated to `enum class` and changed the style for enum values